packages/manager/src/features/Support/SupportTicketDetail/TabbedReply/MarkdownReference.tsx

```diff
@@ -1,66 +1,58 @@
 import * as React from 'react';
-import { makeStyles } from 'tss-react/mui';
 
+import { HighlightedMarkdown } from 'src/components/HighlightedMarkdown/HighlightedMarkdown';
 import { Link } from 'src/components/Link';
+import { Paper } from 'src/components/Paper';
+import { Stack } from 'src/components/Stack';
 import { Typography } from 'src/components/Typography';
 
-import type { Theme } from '@mui/material/styles';
-
-const useStyles = makeStyles()((theme: Theme) => ({
-  example: {
-    backgroundColor: theme.name === 'dark' ? theme.bg.white : theme.bg.offWhite,
-    margin: `${theme.spacing(2)} 0`,
-    padding: theme.spacing(2),
-  },
-  header: {
-    marginBottom: theme.spacing(),
-    marginTop: theme.spacing(2),
-  },
-}));
-
 interface Props {
   isReply?: boolean;
-  rootClass?: string;
 }
 
 export const MarkdownReference = (props: Props) => {
-  const { classes } = useStyles();
-
   return (
-    <div className={props.rootClass}>
+    <Stack spacing={2}>
       <Typography>
         You can use Markdown to format your{' '}
         {props.isReply ? 'reply' : 'question'}. For more examples see this{' '}
```

Comment:
```suggestion
        {props.isReply ? 'reply' : 'question'}. For more examples, see this{' '}
```
Optional since not a style change: add missing comma to copy